core/templates/dev/head/pages/topics_and_skills_dashboard/topics_and_skills_dashboard.html

```diff
@@ -1,2 +1,227 @@
+{% endblock maintitle %}
+
+{% block navbar_breadcrumb %}
+  <topics-and-skills-dashboard-navbar-breadcrumb>
+  </topics-and-skills-dashboard-navbar-breadcrumb>
+{% endblock navbar_breadcrumb %}
+
+{% block local_top_nav_options %}
+  <topics-and-skills-dashboard-navbar>
+  </topics-and-skills-dashboard-navbar>
+{% endblock local_top_nav_options %}
+
+
+{% block content %}
+  <background-banner></background-banner>
+  <div class="oppia-dashboard-container" ng-controller="TopicsAndSkillsDashboard" style="position: relative;">
+    <div>
+      <h2 class="oppia-dashboard-title" translate="I18N_DASHBOARD_TOPICS_AND_SKILLS_DASHBOARD"></h2>
+    </div>
+    <div ng-if="topicSummaries.length > 0 || skillSummaries.length > 0">
+      <ul class="oppia-dashboard-tabs">
+        <li ng-if="topicSummaries.length > 0"
+            ng-class="{'oppia-dashboard-tabs-active': activeTab === 'topics'}">
+          <a class="oppia-dashboard-tabs-text"
+             ng-click="setActiveTab('topics')"
+             translate="I18N_DASHBOARD_TOPICS">
+          </a>
+        </li>
+        <li ng-if="skillSummaries.length > 0"
+            ng-class="{'oppia-dashboard-tabs-active': activeTab === 'skills'}">
+          <a class="oppia-dashboard-tabs-text"
+             ng-click="setActiveTab('skills')"
+             translate="I18N_DASHBOARD_SKILLS">
+          </a>
+        </li>
+      </ul>
+    </div>
+
+    <div class="oppia-card-view-wrap" ng-if="activeTab === 'topics'">
+      <md-card ng-repeat="topic in topicSummaries"
+               class="oppia-activity-summary-tile oppia-dashboard-card-view-item">
+        <a ng-href="/topic_editor/<[topic.id]>">
+          <div class="title-section">
+            <h2 class="activity-title">
+              <span><[(topic.name | truncate:40)]></span>
+            </h2>
+          </div>
+          <div class="content-section">
+              Uncategorized Skills: <[topic.uncategorized_skill_count]>
+          </div>
+          <div class="content-section">
+              Canonical Stories: <[topic.canonical_story_count]>
+          </div>
+          <div class="content-section">
+              Additional Stories: <[topic.additional_story_count]>
+          </div>
+        </a>
+      </md-card>
+    </div>
+
+    <div class="oppia-card-view-wrap" ng-if="activeTab === 'skills'">
+      <md-card ng-repeat="skill in skillSummaries"
+               class="oppia-activity-summary-tile oppia-dashboard-card-view-item">
+        <a ng-href="/skill_editor/<[skill.id]>">
+          <div class="title-section">
+            <h2 class="activity-title">
+              <span><[(skill.description | truncate:40)]></span>
+            </h2>
+          </div>
+          <div class="content-section" style="margin-top: 25%; font-size: 0.9em;">
+              Misconceptions: <[skill.misconception_count]>
+          </div>
+        </a>
+      </md-card>
+    </div>
+  </div>
+  <style>
+  .oppia-dashboard-tabs {
```

Comment:
Indent all these by 2, since it's in an inner scope of the "style" tag.

(Was this a copy-paste issue? If so, would you mind fixing the original source of this pattern too?)

Reply:
Done.